patches/master_patch.patch

```diff
@@ -1827,6 +1827,19 @@ index f1a096e649290ef2e2ce2cc30891f547a6dc196f..1338eb43287476a108454e316dbfb853
    return switches::autoplay::kUserGestureRequiredPolicy;
  #else
    return switches::autoplay::kNoUserGestureRequiredPolicy;
+diff --git a/net/http/BUILD.gn b/net/http/BUILD.gn
+index d00da41da1d8bbc40d8a85d582c1966ed22eaaf1..6a3b54e91eafb8f187989c9c954be0008343aa63 100644
+--- a/net/http/BUILD.gn
++++ b/net/http/BUILD.gn
+@@ -10,7 +10,7 @@ compiled_action("generate_transport_security_state") {
+ 
+   # Inputs in order expected by the command line of the tool.
+   inputs = [
+-    "transport_security_state_static.json",
++    "//electron/atom/browser/resources/transport_security_state_static.json",
```

Comment:
this file should be updated with every Chromium rebase using `transport_security_state_static.json` what's the best process for doing that? maybe an automated script that runs with every rebase?

Reply:
the tricky parts seems to be that we both add and remove entries. If we were only adding them it would be much easier because we could just merge the two files in another gn task. Is it necessary to remove entries?

Reply:
or maybe we could minimize the diff by removing the entries from chrome transport_security_state_static.json as a patch and then merge?

Reply:
I did that before moving `transport_security_state_static.json` to the electron code, it'll be hard to maintain if we do that. I think a separate file makes gives us more control over the pinned domains.

Reply:
@bridiver it's necessary to remove entries because Google told us we shouldn't do certificate pinning for sites we don't have a point-of-contact for in case they break. however, it's probably safe for us to do `force-https` on any site. so @jumde's list removes the `pins` property for non-Brave entries that have it but keeps the `force-https` property

Reply:
@diracdeltas are there any changes that still need to be made here then?

Reply:
@bridiver i think this PR is fine as-is

Reply:
but as-mentioned above, there should be a follow up issue for downstream list maintenance (making sure it happens every browser-laptop release)